sprites.py

```diff
@@ -119,32 +126,36 @@ def insert_in_list(self, spr, i):
         else:
             self.list.insert(i, spr)
 
+    def find_in_list(self, spr):
+        return (spr in self.list)
+
     def remove_from_list(self, spr):
         ''' Remove a sprite from the list. '''
         if spr in self.list:
             self.list.remove(spr)
 
-    def find_sprite(self, pos):
+    def find_sprite(self, pos, region=False):
         ''' Search based on (x, y) position. Return the 'top/first' one. '''
         list = self.list[:]
         list.reverse()
         for spr in list:
```

Comment:
I would preferably like to have any other variable than list, because `list` is a built in function of python, and that would not be good; although it does not cause any errors.